apis/sgv2-quarkus-common/src/main/java/io/stargate/sgv2/api/common/exception/StatusRuntimeExceptionMapper.java

```diff
@@ -39,35 +42,35 @@ public RestResponse<ApiError> statusRuntimeException(StatusRuntimeException grpc
     switch (sc) {
         // Start with codes we know how to handle
       case FAILED_PRECONDITION:
-        return grpcResponse(
+        return grpcResponseNoLog(
             sc,
             Response.Status.BAD_REQUEST,
             "Invalid state or argument(s) for gRPC operation",
             grpcE.getMessage());
       case INVALID_ARGUMENT:
-        return grpcResponse(
+        return grpcResponseNoLog(
             sc,
             Response.Status.BAD_REQUEST,
             "Invalid argument for gRPC operation",
             grpcE.getMessage());
       case NOT_FOUND:
-        return grpcResponse(
+        return grpcResponseNoLog(
             sc, Response.Status.NOT_FOUND, "Unknown gRPC operation", grpcE.getMessage());
       case PERMISSION_DENIED:
-        return grpcResponse(
+        return grpcResponseNoLog(
             sc, Response.Status.UNAUTHORIZED, "Unauthorized gRPC operation", grpcE.getMessage());
       case UNAUTHENTICATED:
-        return grpcResponse(
+        return grpcResponseNoLog(
             sc, Response.Status.UNAUTHORIZED, "Unauthenticated gRPC operation", grpcE.getMessage());
-      case UNAVAILABLE:
+      case UNAVAILABLE: // server-side problem, do log
         // Should this be SERVICE_UNAVAILABLE (503) or BAD_GATEWAY (502)?
```

Comment:
We should be correct wrt to HTTP status code and actually return bad gateway here

Reply:
Done in https://github.com/stargate/stargate/pull/2171/commits/eb812454b4c804ffb27004f518b852ea9e01ed5b